graphql/schemabuilder/reflect.go

```diff
@@ -612,8 +612,8 @@ func (sb *schemaBuilder) buildStruct(typ reflect.Type) error {
 }
 
 var scalars = map[reflect.Type]string{
-	reflect.TypeOf(bool(false)): "bool",
-	reflect.TypeOf(int(0)):      "int",
+	reflect.TypeOf(bool(false)): "Boolean",
+	reflect.TypeOf(int(0)):      "Int",
```

Comment:
I mistakenly interpreted golang's int type to be exactly 32 bits, when it was at least 32 bits:
```
int is a signed integer type that is at least 32 bits in size. It is a distinct type, however, and not an alias for, say, int32.
```
I can revert the change from `int` to `Int`.

As far as mapping all int like types to `Int` and float like types to `Float` goes, I personally prefer to stick with the spec, unless we have a good reason to coerce them into the same type. GraphQL spec also seems to suggest creating a new scalar type if larger than 32 bit integer is required. https://facebook.github.io/graphql/#sec-Int

If we decide to stick with the spec, we'll map golang types `int32`, `float64`, `string`, and `bool` into GraphQL scalar types, and leave other types as it was before.